tools/cardano-sl-tools.cabal

```diff
@@ -140,13 +128,7 @@ executable cardano-wallet-hs2purs
   build-tools: cpphs >= 1.19
   ghc-options: -pgmP cpphs -optP --cpp
 
-  if flag(with-wallet) && flag(with-web)
-    buildable:         True
-  else
-    buildable:         False
-
-  if flag(with-web)
-    cpp-options: -DWITH_WEB
+  buildable:         False
```

Comment:
@gromakovsky I will set `buildable: True`. Let's move `cardano-wallet-hs2purs` with another `PR`, because it is hard to have this PR here in common with `master` (which changes frequently).